Plugin/UIScopeLabel.php

```diff
@@ -0,0 +1,34 @@
+<?php
+namespace AvS\ScopeHint\Plugin;
+
+use Magento\Eav\Api\Data\AttributeInterface;
+use Magento\Framework\Stdlib\ArrayManager;
+use Magento\Catalog\Ui\DataProvider\Product\Form\Modifier\Eav;
+
+class UIScopeLabel
+{
+    public function __construct(
+        protected ArrayManager $arrayManager,
+    ) {
+    }
+
+    public function afterSetupAttributeMeta(Eav $subject, $result, $attribute)
```

Comment:
Can you add types or a docblock for the types of the other parameters and add a return type?

Reply:
Done.

Reply:
Done.